### DISCUSSION TITLE
Stage resolution presets

### DISCUSSION THREAD

**Discussion by BlueDome77** [TurboWarp/extensions#542] [Ideas]
![Screenshot 2023-06-07 at 9 03 08 AM](https://github.com/TurboWarp/extensions/assets/135789443/00806fcc-32b4-4af7-ac29-68fbf799ec27)

**Comment by ghost** [comment]
Yes, I think we need the "actual size of the scene" and the size of the scene in the editor. Sometimes you need a scene that is too large or too small, and this feature will allow you to edit them conveniently.
